### PR TITLE
x64 support and new win32-win64 output directories

### PR DESCRIPTION
**Pull request by robertolarcher**
I'd like to submit some modifications that enable x64 support. Output files, both deliverables (.dll, .lib etc), and intermediates (.obj) are in win32 and win64 directories.

win32win64

win32

x64 dll has been tested on Windows 10 technical preview

### CODE REVIEW

File: win32/readme.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+placeholder for win32 compiled files
```
Comment (sam-github) [inline win32/readme.txt]: Can't the batch files create these output directories?

File: libnet/msvcbuild.bat [hunk 2]
```diff
@@ -1,9 +1,10 @@
-@rem Script to build libnet under "Visual Studio .NET Command Prompt".
```
Comment (sam-github) [inline libnet/msvcbuild.bat]: maybe this file should be renamed `msvcbuild32.bat`?

**Comment by sam-github** [comment]
I'd like to run it before merge, it will take me a couple days before I get some time.

File: win32/readme.txt (hunk 1 above)
Comment (robertolarcher) [inline win32/readme.txt]: Yes, sure. Let me do some tests.

File: libnet/msvcbuild.bat (hunk 2 above)
Comment (robertolarcher) [inline libnet/msvcbuild.bat]: Yes. msvsbuild32.bat would be better.

**Comment by robertolarcher** [comment]
Hi,
any news about this pull?

https://github.com/sam-github/libnet/pull/58

Best regards
Roberto

Date: Mon, 16 Mar 2015 12:13:19 -0700

**Comment by gvanem** [comment]
IMHO, it would be make more sense to put the .obj-files under 
```
<libnet-root>\libnet\src\[Win32|Win64]\[Release|Debug]
```

This seems more like MSbuild does it if a .sln/.vcxproj file was used.

And have your .bat-files name the libraries as:
```
<libnet-root>\libnet\lib\[x86|x64]\libnet*.lib
<libnet-root>\libnet\lib\[x86|x64]\libnet*.dll
```

But the .bat-files are not very nice. I.e. they breaks e.g. a MingW build by copying `<stdint.h>` into the include-path. How about an `#ifdef _MSC_VER` inside `libnet.h` instead?

**Comment by troglobit** [comment]
Long time no progress on this PR.  This was referenced in #65, which has now been merged so I'm closing this PR. If there are any outstanding issues, please submit a new PR.

**Pull request closed without merging**